engine/table/src/main/java/io/deephaven/engine/table/impl/select/WhereFilterImpl.java

```diff
@@ -18,4 +18,9 @@ public boolean isAutomatedFilter() {
     public void setAutomatedFilter(boolean value) {
         isAutomatedFilter = value;
     }
+
+    @Override
+    public final <T> T walk(Visitor<T> visitor) {
+        return visitor.visitOther(this);
+    }
```

Comment:
Defaulting to `visitOther` seems like it would be nice for 3rd party `WhereFilter` implementors.